.circleci/config.yml

```diff
@@ -1,17 +1,35 @@
 version: 2
+
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:8.11
+
 workflows:
   version: 2
   build-and-test:
     jobs:
       - build
+  build-deploy:
+    jobs:
+      - build:
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
Why are we building only the tags and not the rest of the branches?

Reply:
Because this workflow is 'build and deploy'. It's activated by a tag creation. It builds the source and publishes on npm.
The other workflow, 'build-and-test' runs for each commit.

Reply:
It was 6 am, I was asleep didn't see the build-and-test